<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>668</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>620</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Open</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>741</width>
     <height>581</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="PM">
    <attribute name="title">
     <string>PM</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PM Numbers/Range</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="procedure">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MECH PM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ELEC PM</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="workType">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>IN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>REP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RM</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="priority">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FOR REVIEW</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HS</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="data_division">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Procedure</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Work Type</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Data Division</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>20</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Priority</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate WO Based On</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="generate1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>95</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>On Due</string>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">generate</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="generate2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>320</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>On Complete</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">generate</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="generate3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>320</y>
       <width>95</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>All</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">generate</string>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Inactive PM?</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="inactive1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>95</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Yes</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">inactive</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="inactive2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>390</y>
       <width>95</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>No</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">inactive</string>
     </attribute>
    </widget>
    <widget class="QLineEdit" name="pmNum">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Either formatted like &quot;1,2,3,4&quot; or &quot;1-4&quot;.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Equipment">
    <attribute name="title">
     <string>Equipment</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="generate">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="inactive">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
